docs/README.md

```diff
@@ -15,7 +15,7 @@ Forge is a free, open-source modding API all of your favourite mods use!
 * [Discord]
 * [Documentation]
 
-# Installing NeoForged
+# Installing NeoForge
 
 Go to [CurseForge][CurseForge] project, select the minecraft version and installer, and run it.
```

Comment:
Why does this even mention CurseForge? Download from the main page at https://neoforged.net or the projects link therein.

Reply:
Fixed with latest commit removing curseforge section and adding link to neoforge download